### ISSUE TITLE
Not working within admonitions and indented (block) text

### ISSUE THREAD

**Issue by hug-sch** [tobiasah/mkdocs-caption#6]
The plugin numbering and class inserting doesn't seem to work within an admonition (!!! note for example; text after that should also be indendented) or within the indented text of, for example a numbered list.

**Comment by tobiasah** [comment]
Thanks for bringing this up. Can you provide an example for both? I don`t think I fully understand. 

Because this works for me:
```markdown
!!! warning

    [](#my_id) demonstrates ...
```

**Comment by hug-sch** [comment]
Hi Tobias, the problem is with the Figure: caption. For example:
```
!!! note
      testing
      
      Figure: Mycaptionl.{class=pp-figure}

      ![](../images/....png)
```
uses the caption as literal text, does not number the figure and displays also the text {class= ...}
I think it has something to do with the indention because this also occurs with indented lists.

**Comment by hug-sch** [comment]
Any news about this?

**Comment by hug-sch** [comment]
Any news?

**Comment by tobiasah** [comment]
Sorry, the last weeks were quite busy. I released a new version 0.0.10 which should fix the issue.
